src/lib/components/Modal/Modal.jsx

```diff
@@ -46,8 +47,8 @@ const preRender = (
 
   return (
     <div
+      {...transferProps(restProps)}
```

Comment:
According to you, is it ok to leave `transferProps` on the root backdrop `div`? I couldn't decide whether to use it as it is or to use it in child `div` for modal.

Reply:
We decided collectively on our meeting to move it to the child div that represents modal window.